extensions.json

```
@@ -793,9 +797,7 @@
     },
     {
       "id": "lextudio.restructuredtext",
-      "repository": "https://github.com/vscode-restructuredtext/vscode-restructuredtext",
-      "version": "143.0.0",
-      "checkout": "143.0.0"
+      "repository": "https://github.com/vscode-restructuredtext/vscode-restructuredtext"
```

Not sure why the `version` & `checkout` got removed:

```suggestion
      "repository": "https://github.com/vscode-restructuredtext/vscode-restructuredtext",
      "version": "145.0.0",
      "checkout": "145.0.0"
```